README.rst

```diff
@@ -440,10 +438,10 @@ Driver specific installation instructions can be found on Read The Docs:
 * `nidcpower <http://nimi-python.readthedocs.io/en/master/nidcpower.html#installation>`_
 * `nidmm <http://nimi-python.readthedocs.io/en/master/nidmm.html#installation>`_
 * `nifgen <http://nimi-python.readthedocs.io/en/master/nifgen.html#installation>`_
-* `nimodinst <http://nimi-python.readthedocs.io/en/master/nimodinst.html#installation>`_
 * `niscope <http://nimi-python.readthedocs.io/en/master/niscope.html#installation>`_
-* `nise <http://nimi-python.readthedocs.io/en/master/nise.html#installation>`_
 * `niswitch <http://nimi-python.readthedocs.io/en/master/niswitch.html#installation>`_
+* `nise <http://nimi-python.readthedocs.io/en/master/nise.html#installation>`_
+* `nimodinst <http://nimi-python.readthedocs.io/en/master/nimodinst.html#installation>`_
```

Comment:
These are all unrelated changes from previous PRs. Please merge latest master.

Reply:
Based on @sbethur 's comment below, I think it is correct to include this change?

Reply:
@lkolbly, I'm creating a PR to fix this. Please exclude this file from your PR.

Reply:
@sbethur Pushed a commit that should revert it.